games-util/pyfa/pyfa-1.35.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit eutils desktop python-r1 python-utils-r1
```

Comment:
Please sort lexically whenever possible, and don't inherit `python-utils-r1` directly.